src/Common/tests/System/Net/RemoteServerQuery.cs

```diff
@@ -0,0 +1,79 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Test.Common
+{
+    internal static class RemoteServerQuery
+    {
+        internal static async Task Run(Func<Task> testCode, Type remoteServerFailExceptionType, string servername, bool mustFail = true)
+        {
+            try
+            {
+                await testCode();
+            }
+            catch (Exception actualException)
+            {
+                if (remoteServerFailExceptionType.Equals(actualException.GetType()))
+                {
+                    if (mustFail)
```

Comment:
coding style: use braces in 'if' clauses.

Reply:
Not required, see [here](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)

```
We use Allman style braces, where each brace begins on a new line. A single line statement block can go without braces but the block must be properly indented on its own line and it must not be nested in other statement blocks that use braces (See issue 381 for examples).
```